starwars/src/App.css

```diff
@@ -5,4 +5,46 @@
   color: #443e3e;
   text-shadow: 1px 1px 5px #fff;
 }
+
+.charWrapper {
+  height: 90vh;
+  width: 100%;
+  display: grid;
+  justify-content: center;
+}
+
+.charMap {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: repeat(auto-fit, minmax(200px, 1fr));
+  grid-gap: 5px;
+}
+
+
+.chars{
+  border: 5px solid darkolivegreen;
+  border-radius: 15px;
+  width: 100%;
+  text-align: center;
+  background-color: sandybrown;
+  opacity: 0.5;
```

Comment:
I really like the opacity, but I would recommend that you put the opacity only on the background so your text is still readable. You can do this by setting the background color to an rgba value (for example, rgba(244, 164, 96, 0.5))